src/Matches.js

```diff
@@ -82,42 +82,49 @@ class Matches extends React.Component {
         
     }
 
-
     render(){
         let matchingSection;
         if (this.state.matchesExist == "exists"){
             let userItems = [];
             for (var i = 0; i < this.state.userIds.length; i++){
                 var pos_user = this.state.userIds[i];
                 userItems.push(
-                    <button className='pos-user' key={pos_user} value = {this.state.userIds[i]+"|"+this.state.firstnames[i]} onClick = {this.selectUser}>
+                    <button className='pos-user' 
+                            key={pos_user} 
+                            value = {this.state.userIds[i]+"|"+this.state.firstnames[i]} 
+                            onClick = {this.selectUser}>
                         {this.state.firstnames[i]}
-                    </button>
+                    </button>  
                 )
+                
+
             }
-            matchingSection = 
-                <div>
-                    <td id='Matches-congrats'>
-                        Congratulations, these people want to be your friend!
-                    </td>
+            matchingSection = <h3 id='Matches-congrats'>
+                <p>Congratulations,</p>
+                <p>You got potential pal-ed!</p> 
+                <p>
                     {userItems}
-                </div>
+                </p>
+                </h3>
         }
         else if (this.state.matchesExist == "not exists") {
             matchingSection = <h2 id='Matches-none'>
-                Sorry, No one wants to be your friend for now. Please try again later :(
+                <p>Sorry, no one met the matching criteria you set.</p> 
+                <p>We suggest you to edit your profile.</p> 
+                <p>Please try again later :(</p>
             </h2>
         }
         else {
             matchingSection = <h2></h2>
         }
         return(
             <div id='Matches-section'>
-                <h2 id='Matches-header'>Matches</h2>
-                <div>{this.state.name}</div>
-                <Link to={{pathname: '/main', state: {id: this.state.userId}}}>Back to Swiping!</Link>
                 <div id='Matches-section'>
                     {matchingSection}
+                </div> 
+                <button id = 'messagingButton'>Say Hello</button> 
```

Comment:
did you test this to make sure it works? Also does this specify what user you are trying to say hello to